### DISCUSSION TITLE
Help ReadTheDocs to better integrate with MkDocs

### DISCUSSION THREAD

**Discussion by pawamoy** [mkdocs/mkdocs#3413] [General]
MkDocs 1.5 added a custom `!relative` tag. ReadTheDocs currently only handles `!!python/name` tags. Support was added in this PR: https://github.com/readthedocs/readthedocs.org/pull/7507. Latest code can be found here: 
https://github.com/readthedocs/readthedocs.org/blob/main/readthedocs/doc_builder/backends/mkdocs.py. As you can see the safe loader still only handles these tag. I guess it means that `!relative` tag are not supported, and upon re-dumping YAML, these values would simply be lost.

ReadTheDocs wants to find a way to better integrate with MkDocs, to avoid having to fake-implement custom tags just to inject configuration. See https://github.com/readthedocs/readthedocs.org/issues/7844. Maybe we could discuss this here as well.

**Comment by humitos** [comment]
We are merging and deploying this PR today/tomorrow: https://github.com/readthedocs/readthedocs.org/pull/11206 that removes the manipulation of the YAML file from Read the Docs.

Users should be able to use all the supported YAML syntax on Read the Docs after the deploy 🎉 . Thanks for all your support!

**Reply by pawamoy** [reply]
Great news @humitos! Congrats to the RTD team 🎉

**Comment by humitos** [comment]
Can this discussion be marked as solved now that people can use all the YAML syntax that MkDocs supports? Thanks.
